app/services/data_migrations/backfill_sites_from_temp_sites.rb

```diff
@@ -0,0 +1,15 @@
+module DataMigrations
+  class BackfillSitesFromTempSites
+    TIMESTAMP = 20220620162528
+    MANUAL_RUN = false
```

Comment:
could we run this as manual true, as this might take long on envs to backfill the data